code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -1,9 +1,25 @@
+#define OVERLAY_FIRE_LAYER -1
+
 /mob/living/simple_animal
 	name = "animal"
 	icon = 'icons/mob/animal.dmi'
 	health = 20
 	maxHealth = 20
 
+	speed = 0 //LETS SEE IF I CAN SET SPEEDS FOR SIMPLE MOBS WITHOUT DESTROYING EVERYTHING. Higher speed is slower, negative speed is faster
+
+	//LETTING SIMPLE ANIMALS ATTACK? WHAT COULD GO WRONG. Defaults to zero so Ian can still be cuddly
```

Comment:
codedocs go before the comment, https://github.com/cmss13-devs/cmss13/blob/master/.github/guides/AUTODOC.md